tests/protocol/test_wire.py

```diff
@@ -218,10 +214,9 @@ def test_get_host_ga_plugin(self, *args): # pylint: disable=unused-argument
     def test_upload_status_blob_should_use_the_host_channel_by_default(self, *_):
         def http_put_handler(url, *_, **__): # pylint: disable=inconsistent-return-statements
             if protocol.get_endpoint() in url and url.endswith('/status'):
-                return MockResponse(body=b'', status_code=200)
+                return MockResponse(body=b'OK', status_code=200)
```

Comment:
I see `b'OK'` being used in multiple places, why not make it a constant of the `MockResponse` class?